ng-dev/commit-message/validate-range/validate-range.ts

```diff
@@ -43,12 +46,14 @@ export async function validateCommitRange(from: string, to: string) {
             .filter(isNonFixup)
             .map(extractCommitHeader),
     };
-    const {valid, errors: localErrors} = validateCommitMessage(commit, options);
+
+    const {valid, errors: localErrors} = await validateCommitMessage(commit, options);
     if (localErrors.length) {
       errors.push([commit.header, localErrors]);
     }
-    return valid;
-  });
+
+    allCommitsInRangeValid = allCommitsInRangeValid && valid;
```

Comment:
We don't need to continue checking after we find an invalid commit.  Once we find an invalid commit, we can set it to false and `break`

Reply:
The way the code was written, it seemed like we actually would want to collect as much findings as possible, instead of printing them sequentially one by one after fixing

Reply:
I will keep it like that because otherwise keeping track of the errors that way wouldn't make sense as well. Happy to clean up in a follow-up.

Reply:
Makes sense, its probably best that we collect all of the findings.  From the perspective of how it was previously written `Array.every` exits as soon as it encounters a `false`.  But as you said tracking all of the errors is probably what we would really like.

Reply:
Yeah, agreed. I realized `every` would exit early, but then it felt like everything around it was built to track more errors. could have been a simple `.find` otherwise